patches/minecraft/net/minecraft/tileentity/TileEntity.java.patch

```diff
@@ -31,15 +31,30 @@
     }
  
     public CompoundNBT func_189515_b(CompoundNBT p_189515_1_) {
-@@ -62,6 +_,8 @@
-          p_189516_1_.func_74768_a("x", this.field_174879_c.func_177958_n());
-          p_189516_1_.func_74768_a("y", this.field_174879_c.func_177956_o());
-          p_189516_1_.func_74768_a("z", this.field_174879_c.func_177952_p());
-+         if (this.customTileData != null) p_189516_1_.func_218657_a("ForgeData", this.customTileData);
-+         if (getCapabilities() != null) p_189516_1_.func_218657_a("ForgeCaps", serializeCaps());
-          return p_189516_1_;
+@@ -54,15 +_,18 @@
+    }
+ 
+    private CompoundNBT func_189516_d(CompoundNBT p_189516_1_) {
++      CompoundNBT compoundNBT = p_189516_1_.func_74737_b();
```

Comment:
Not sure about this PR overall (didn't really review the functionality or things) but wouldn't this be a lot smaller of a patch if it was just `p_189516_1_ = p_189516_1_.func_74737_b()`